tools/serverpod_cli/test/generator/dart/client_code_generator/client_test.dart

```diff
@@ -133,4 +133,83 @@ void main() {
               'caller.callStreamingServerEndpoint<Future<String>, String'));
     });
   });
+
+  group(
+      'Given a protocol definition with a method with Deprecated(..) annotation when generating client file',
```

Comment:
Suggestion(NIT): Enclose the deprecated keyword with quotation marks to make the description a bit more clear.
```suggestion
      'Given a protocol definition with a method with "@Deprecated(..)" annotation when generating client file',
```